README.md

```diff
@@ -10,6 +10,24 @@ It is composed of:
 
 The processing modules can either be run as standalone code or asynchronously as Celery tasks.
 
+---
+# Overal table for showing the usage of short-form of arguments of all CLIs individually
+| Argument short form | Download CLI            | Copy CLI
+| ------------------- | --------------------    | --------------
+| '-d'                | '--down_dir'            | '--destination_path'
```

Comment:
```suggestion
| '-d'                | '--destination_dir'            | '--destination_dir'
```
Also move adding the `destination_dir` argument to `util.parse_common_args`

Reply:
`down_dir` concept is different from `destination_path` concept, so they can not be merged into one common thing! If we want to merge, then what should I right for in the help of the argument so that the users of both CLIs are not confused? 

We also need to change some other files (templates, configurations) if we want to change the name! I think it is better not to merge them because it becomes obscure and may become confusing.

Reply:
The concept is exactly the same. `down_dir` is where you downloaded file will end up, right? That's what is called destination.
Please change the name.